governance-exercises/Le Bateleur - Interpretation Exercise 01/GOV 12 Rewrite.md

```diff
@@ -0,0 +1,56 @@
+# Interpretation Exercise 01
+
+---
+
+## Proposed Rewrite of GOV 12
+
+To redress the aforementioned concerns, I would rewrite GOV 12 to achieve three aims. 
+
+First, I create a general protocol for GOV 12 and restrict its application to "substantial threats" to the functioning of Maker Governance. This is a far higher showing to meet than the current one. It redresses the risk stemming from ambiguity/overbreadth, while preserving to a degree the original "bootstrapping" intention.
+
+Second, I require endorsements from rotating ecosystem actors to provide checks/balances against the concentration of power in Governance Facilitators.
+
+Third, I carve out exceptions to the general protocol for 1) clear mistakes and 2) Launch Project-related needs. The former exclusion for clear mistakes somewhat preserves the original intention behind GOV 12, but in a narrowly tailored way to better guard against misalignment. The latter exclusion is appropriate, I believe, because the [Launch Project](https://mips.makerdao.com/mips/details/MIP108#9-launch-project) is literally framed as an exception to the rule of maximum transparency in governance processes. Further, the single instance of GOV 12 governance practice was the introduction of the EDSR.  In [Discord](https://discord.com/channels/893112320329396265/899667119426371625/1131216377575919716) as well as the [Forum proposal](http://forum.makerdao.com/t/request-for-gov12-1-2-edit-to-the-stability-scope-to-quickly-implement-enhanced-dsr/21405/6?u=le_bateleur), Rune maintained the EDSR was necessary to stoke momentum for the Launch Project (Phase 1). 
```

Comment:
Separating out clear mistakes is good, imo. Makes that set of criteria much more clear, and easier to apply.

I'm not sure whether EDSR would fall under the launch project to be honest. I guess Rune made that argument, but not sure I really bought it. Why do we need momentum within a week rather than within a month, etc.